charts/soroban-rpc/values.yaml

```diff
@@ -32,36 +32,78 @@ global:
 sorobanRpc:
   ## See global.image.sorobanRpc for image settings
 
+
   ## This variable allows overrides of default settings. It will be used as a container port
   ## and port where soroban-rpc server will listen.
+  # port: 8000
+
+  ## Soroban-rpc config variables
   sorobanRpcConfig:
-    port: 8000
-    ingress:
-      host: example.com
-      annotations:
-        kubernetes.io/ingress.class: "public"
-        cert-manager.io/cluster-issuer: "default"
+    endopoint: "0.0.0.0:8000"
```

Comment:
would it be good to enable rpc's `admin_endpoint: "0.0.0.0:6061"` ? since it's a statefulset, users could reliably access it on cluster from `<common.fullname>-1` pod, or for consideration, the chart could have something like 'publish-admin'  flag, where it enables it and exposes it on service/ingress?

Reply:
Yeah that's true. I'm wondering actually - do we ever want the endpoint to be anything else than `0.0.0.0:8000` and admin_port, if enabled, `0.0.0.0:6061`?

In order to expose container ports we need to know port numbers which complicates things. Maybe it makes sense to hardcode endpoint to `0.0.0.0:8000` and always expose port 8000?
For the admin port we could add `enableAdminPort: true|false` or similar. If set we could use `0.0.0.0:6061` in the config and port `6061` when exposing container ports. WDYT?

Reply:
yes, for convention rather than config, think default both `0.0.0.0:8000` and `0.0.0.0:6061` on rpc container spec, and have a service map on 8000 and skip need for `enableAdminPort: true|false` chart param. Then it's up to user how they want to access admin port from container. For metrics, could probably just document to change their prom config in cluster to scrape `<common.fullname>-1:6061\metrics`

Reply:
I added example prometheus scrape annotation, those with prometheus deployed to k8s should be able to use it to scrape metrics.